src/streaming/models/VideoModel.js

```diff
@@ -51,6 +51,14 @@ function VideoModel() {
         stalledStreams = [];
     }
 
+    function setup() {
+        eventBus.on(Events.PLAYBACK_PLAYING, onPlaying, this);
```

Comment:
in the dash-If reference player, the videoModel is created when attachView function is called (when the html page is loaded). So, when the user switch to a second stream a call to videoModel.reset will be done, that unregistered PLAYBACK_PLAYING...am I wrong?

Reply:
Not wrong at all! Thanks! The lifetime of the video model is a little messy